pyracf/common/irrsmo00.py

```diff
@@ -21,7 +21,7 @@ def __init__(self) -> None:
 
     def call_racf(self, request_xml: bytes, precheck: bool = False) -> str:
         """Make request to call_irrsmo00 in the cpyracf Python extension."""
-        options = 11 if precheck else 9
+        options = 15 if precheck else 13
```

Comment:
What was this change made for?

Reply:
This was to activate the "Fail on first" error flag, that I had hoped would have fixed add without adding the preliminary extract. It does not accomplish this, and may be unnecessary as, to my knowledge, current pyracf is incapable of generating a "multi-command" xml to send to IRRSMO00, but it doesn't impact our functionality, so it feels safer to leave it in just in case that can happen.

Reply:
Thank you for the explanation.